### PR TITLE
Added login page and login firebase integration

### PR DESCRIPTION
**Pull request by MrClean-1**

### CODE REVIEW

File: code/src/pages/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            />
+            <IonInput
+                type="password"
+                placeholder="Password"
+                className="password loginInputs"
+                onIonChange={(e: any) => setPassword(e.target.value)}
+            />
+            <IonButton
+                className="loginButton"
+                onClick={async () => await handleSubmit(username, password)}>
```
Comment (lemohammed) [inline code/src/pages/Login.tsx:61]: No need for await here :)

File: code/src/pages/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+/**
+ * @author Carter Zimmer & Dennis Lieu
+ * @constructor sets up the state variables that get bound to the input fields and
+ * are used when the login button is clicked
+ */
+const Login: React.FC = () => {
+  const [username, setUsername] = React.useState("");
+  const [password, setPassword] = React.useState("");
```
Comment (lemohammed) [inline code/src/pages/Login.tsx:35]: You could add useState to your imports

File: code/src/pages/Login.tsx (hunk 1 above)
Comment (MrClean-1) [reply code/src/pages/Login.tsx:61]: tysm

File: code/src/pages/Login.tsx (hunk 2 above)
Comment (MrClean-1) [reply code/src/pages/Login.tsx:35]: Yup changing rn

File: code/.eslintrc.json [hunk 3]
```diff
@@ -1,23 +0,0 @@
-{
```
Comment (lemohammed) [inline code/.eslintrc.json:1]: This file should not be deleted otherwise your build will fail

Comment (MrClean-1) [reply code/.eslintrc.json:1]: That'll do it. Not sure why that happened

**Comment by lemohammed** [comment]
How about you rebase onto my branch and I close my pull request?

**Comment by MrClean-1** [comment]
Yeah I'll do that rn

**Comment by RatikKapoor** [comment]
@MrClean-1 can you create a new branch off main and manually apply these commits. With all the cleanup rebase got really messed up. I'm gonna close this PR since it'll be impossible to clean merge

**Pull request closed without merging**